Services/RefundInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MollieShopware\Services;
+
+use Mollie\Api\Resources\Order as MollieOrder;
+use Mollie\Api\Resources\OrderLine;
+use Mollie\Api\Resources\Payment as MolliePayment;
+use MollieShopware\Models\Transaction;
+use Shopware\Models\Order\Detail;
+use Shopware\Models\Order\Order;
+
+interface RefundInterface
+{
+    public function refundOrderAmount(Order $order, Transaction $transaction, float $customAmount = null);
```

Comment:
just a big thank you for an interface :)

question though:
is this the old implementation or your new one?
it seems as if some things our duplicated like processRefund...wouldnt that be more a private thing because we already have a few public refunds?
also i'd be happy if we could remove optional params such as $quantity=1, $customAmount=null, it easily leads to errors and different behaviours of a method.
...in case its really a new code, if its an old one we can keep it for now to avoid breaking anything

Reply:
it's the old code from the backend controller, I think :)